tests/test_manager.py

```diff
@@ -309,6 +311,24 @@ def test_url_for(self):
             assert url3.endswith('/api/people/1/articles')
             assert url4.endswith('/api/people/1/articles/2')
 
+    def test_url_for_explicitly_sets_primary_key_in_links(self):
+        """Should use the primary_key explicitly set when generating links"""
+        article = self.Article(id=1, title=u'my_article')
+        self.session.add(article)
+        self.session.commit()
+        self.manager.create_api(self.Article, primary_key='title')
+
+        response = self.app.get('/api/article')
+        document = loads(response.data.decode('utf-8'))
+        articles = document['data']
+        article = articles[0]
+
+        assert 'my_article' in article['links']['self']
+        assert '/1' not in article['links']['self']
+        author_links = article['relationships']['author']['links']
+        assert author_links['self'] == (
```

Comment:
Should this be `author_links['self'].endswith(...)`? In other words, doesn't the link start with `http://example.com/...`?